### DISCUSSION TITLE
Achievements

### DISCUSSION THREAD

**Discussion by AdamW1998** [community/community#20580] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everyone, I want some information about these achievements. Where can I get this?

**Comment by Mehul2203** [comment, marked as answer]
Hey @AdamW1998 ,
check this:
[https://github.com/Schweinepriester/github-profile-achievements](https://github.com/Schweinepriester/github-profile-achievements)

**Comment by DhanushNehru** [comment]
Hi Everyone, It seems github achievements is not getting updated or not working as expected, can anyone help ?

**Reply by Mehul2203** [reply]
Hello @DhanushNehru ,
Sure, I can help, could you please elaborate on the problem you're facing?
Thanks!

**Reply by DhanushNehru** [reply]
Hi @Mehul2203 

For example, in my profile star achievement, I have another repo which has more stars but not listed while clicking on it, same repo is repeated twice

Similarly other merge pull requests etc is not updated

**Reply by Mehul2203** [reply]
@DhanushNehru, as I can see your "Starstruck" achievement on your profile, is at the **Bronze Level** because of one of your repositories with the maximum stars **358** (https://github.com/DhanushNehru/Ultimate-NodeJs-Resources). This achievement will be updated to "Silver Level" when any one of your repositories has **512 starts** (GitHub does not add up all of the stars of each repository, it only upgrades the achievement based on one repository). 
Please refer to this achievement repository: https://github.com/Schweinepriester/github-profile-achievements

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @AdamW1998. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
